services/doctor/doctor.go

```diff
@@ -94,6 +97,14 @@ func RunChecks(ctx context.Context, colorize, autofix bool, checks []*Check) err
 			}
 			dbIsInit = true
 		}
+		if !storageIsInit && check.InitStorage {
```

Comment:
@KN4CK3R Should i send a seperate PR for this?

Reply:
Probably a good idea

Reply:
- 28921